From 57a0658d87bb74fbc4d62815e2f8ae5fc31c764e Mon Sep 17 00:00:00 2001
From: Mark Vainomaa <mikroskeem@mikroskeem.eu>
Date: Fri, 10 May 2024 12:21:11 +0300
Subject: [PATCH] Remove muldefs linker flag

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 8d71579a..e8e84777 100644
--- a/Makefile
+++ b/Makefile
@@ -68,7 +68,7 @@ ifeq (cleveldb,$(findstring cleveldb,$(COSMOS_BUILD_OPTIONS)))
   ldflags += -X github.com/cosmos/cosmos-sdk/types.DBBackend=cleveldb
 endif
 ifeq ($(LINK_STATICALLY),true)
-  ldflags += -linkmode=external -extldflags "-Wl,-z,muldefs -static"
+  ldflags += -linkmode=external -extldflags "-static"
 endif
 ifeq (,$(findstring nostrip,$(COSMOS_BUILD_OPTIONS)))
   ldflags += -w -s
-- 
2.44.0

