From 39f9894af30373a6bcf8c279b54bd9d99c289f91 Mon Sep 17 00:00:00 2001
From: Mark Vainomaa <mikroskeem@mikroskeem.eu>
Date: Thu, 28 Dec 2023 13:18:03 +0200
Subject: [PATCH] Create build task

---
 Makefile | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/Makefile b/Makefile
index 0e86d8be..43cfac90 100644
--- a/Makefile
+++ b/Makefile
@@ -95,6 +95,11 @@ install: go.sum
 		@go install -mod=readonly $(BUILD_FLAGS) ./cmd/zetaclientd
 		@go install -mod=readonly $(BUILD_FLAGS) ./cmd/zetaclientd-supervisor
 
+build: go.mod
+		go build -mod=readonly $(BUILD_FLAGS) -o bin/zetacored ./cmd/zetacored
+		go build -mod=readonly $(BUILD_FLAGS) -o bin/zetaclientd ./cmd/zetaclientd
+		go build -mod=readonly $(BUILD_FLAGS) -o bin/zetaclientd-supervisor ./cmd/zetaclientd-supervisor
+
 install-zetaclient: go.sum
 		@echo "--> Installing zetaclientd"
 		@go install -mod=readonly $(BUILD_FLAGS) ./cmd/zetaclientd
-- 
2.44.0

